neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -291,6 +291,11 @@ private void OnVersionMessageReceived(VersionPayload payload)
             Context.Parent.Tell(payload);
         }
 
+        private void OnDisconnectMessageRecived(DisconnectPayload payload)
```

Comment:
```suggestion
        private void OnDisconnectMessageReceived(DisconnectPayload payload)
```
minor typo in "Received"

Reply:
tks, fixed.